ansible_deployer/command_line.py

```diff
@@ -200,15 +200,51 @@ def check_cfg_permissions_and_owner(cfg_path: str):
         logger.error("Program will exit now.")
         sys.exit(41)
 
+def get_config_paths():
+    """Function to create absolute config paths and check their extension compatibility"""
+    ymls = []
+    yamls = []
+    infra_cfg = None
+    tasks_cfg = None
+
+    for config in os.listdir(conf["global_paths"]["config_dir"]):
+        if config != "ansible-deploy.yaml":
+            if config.endswith(".yml"):
+                ymls.append(config)
+            elif config.endswith(".yaml"):
+                yamls.append(config)
+
+            if config.startswith("infra"):
+                infra_cfg = os.path.join(conf["global_paths"]["config_dir"], config)
+            elif config.startswith("tasks"):
+                tasks_cfg = os.path.join(conf["global_paths"]["config_dir"], config)
+
+    if len(ymls) > 0 and len(yamls) > 0:
+        logger.critical("Config extensions are not compatible!")
```

Comment:
This error message should tell what this "compatibility" mean. It's better to say. All files needs to have the same extension either yml or yaml - such a communicate after debug messages with lists of files is something easy to understand.